CMake/SlicerApplicationOptions.cmake

```diff
@@ -114,3 +114,29 @@ if(NOT DEFINED ${Slicer_MAIN_PROJECT_APPLICATION_NAME}_SOURCE_DIR)
   set(${Slicer_MAIN_PROJECT_APPLICATION_NAME}_SOURCE_DIR ${CMAKE_SOURCE_DIR})
 endif()
 mark_as_superbuild(${Slicer_MAIN_PROJECT_APPLICATION_NAME}_SOURCE_DIR)
+
+#-----------------------------------------------------------------------------
+# Set default installation folder and admin account requirement for Windows
+#-----------------------------------------------------------------------------
+if(WIN32)
+
+  if(NOT DEFINED Slicer_CPACK_NSIS_INSTALL_ROOT)
+    # Set root install directory (displayed to end user at installer-run time)
+    # to C:\Users\<username>\AppData\Local by default to allow installation
+    # without having administrative privileges.
+    set(Slicer_CPACK_NSIS_INSTALL_ROOT "$LOCALAPPDATA" CACHE STRING
```

Comment:
let's set type to `PATH`. See https://cmake.org/cmake/help/latest/command/set.html#set-cache-entry